ci-operator/step-registry/gcp/provision/minimal-permission/gcp-provision-minimal-permission-commands.sh

```diff
@@ -26,7 +30,44 @@ fi
 # The IAM service account for IPI: ipi-min-permissions-sa@${GOOGLE_PROJECT_ID}.iam.gserviceaccount.com
 # The IAM service account for UPI: upi-min-permissions-sa@${GOOGLE_PROJECT_ID}.iam.gserviceaccount.com
 # Currently we only deal with IPI in Prow CI.
-iam_account="ipi-min-permissions-sa@${GOOGLE_PROJECT_ID}.iam.gserviceaccount.com"
 
-gcloud iam service-accounts keys create "${SHARED_DIR}/gcp_min_permissions.json" --iam-account="${iam_account}" || exit 1
-echo "$(date -u --rfc-3339=seconds) - Created a temporary key of the IAM service account for the minimum permissions testing on GCP."
+if [[ "${MINIMAL_PERMISSIONS_WITHOUT_ACT_AS}" == "no" ]]; then
+  iam_account="ipi-min-permissions-sa@${GOOGLE_PROJECT_ID}.iam.gserviceaccount.com"
```

Comment:
Based on we already start to use sa json from `CLUSTER_PROFILE_DIR`, we do not need to reply on the iam account any more which is pre-configured in project, right ?